code/mobility-services/mobility-services.css

```diff
@@ -340,36 +431,38 @@ a.small-button {
   align-items: flex-start;
   flex-direction: row;
   justify-content: center;
-  font-family:'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
-  break-before:page;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
+    "Lucida Sans", Arial, sans-serif;
+  break-before: page;
 }
 
 .image-seal {
-  content:url("https://www.austintexas.gov/sites/default/files/images/Transportation/Home_Page_Gallery/2023%20Austin%20Transportation%20Public%20Works%20Branding%20Guide%2004.28.23_Color%20stack.png");
-  width: 152px; 
+  content: url("https://www.austintexas.gov/sites/default/files/images/Transportation/Home_Page_Gallery/2023%20Austin%20Transportation%20Public%20Works%20Branding%20Guide%2004.28.23_Color%20stack.png");
```

Comment:
Your reference URL here for the city seal is from a non managed source, if it ever changes it will break. Save the image in the github repository if it doesnt exist or you can save to gitbook which I use for images, videos, spreadsheets, & documents.